go/service/main.go

```diff
@@ -1439,3 +1439,11 @@ func (d *Service) StartStandaloneChat(g *libkb.GlobalContext) error {
 
 	return nil
 }
+
+func assertLoggedIn(ctx context.Context, g *libkb.GlobalContext) error {
```

Comment:
there are two `assertLoggedIn` in `teams.go` and ` kvstore.go`, I wonder if we can move them to BaseHandler and use it instead

Reply:
basehandler is not contextified. but I can refator teams and kvstore to use this fn

Reply:
sounds good, thank you!

Reply:
donezo